src/anemoi/graphs/plotting/displots.py

```diff
@@ -0,0 +1,84 @@
+import logging
+from pathlib import Path
+from typing import Optional
+from typing import Union
+
+import matplotlib.pyplot as plt
+from torch_geometric.data import HeteroData
+
+from anemoi.graphs.plotting.prepare import _get_edge_attribute_dims
+from anemoi.graphs.plotting.prepare import _get_node_attribute_dims
+
+LOGGER = logging.getLogger(__name__)
+
+
+def plot_dist_node_attributes(graph: HeteroData, out_file: Optional[Union[str, Path]] = None) -> None:
```

Comment:
Those two functions are quite similar and hence there is a bit of repeated code across them. I think it could be easier to define base function to plot_distributions that could be reused to plot both nodes and edges distributions.